x/emissions/keeper/inference_synthesis/network_regrets.go

```diff
@@ -173,186 +178,199 @@ func GetCalcSetNetworkRegrets(
 	}
 
 	// R^-_ij - Naive Regrets
-	for _, infererLoss := range networkLosses.InfererValues {
-		lastRegret, _, err := k.GetNaiveInfererNetworkRegret(ctx, topicId, infererLoss.Worker)
+	for _, infererLoss := range sortedInfererValues {
+		lastRegret, _, err := args.K.GetNaiveInfererNetworkRegret(args.Ctx, args.TopicId, infererLoss.Worker)
 		if err != nil {
 			return errorsmod.Wrapf(err, "failed to get inferer regret")
 		}
 		newInfererRegret, err := ComputeAndBuildEMRegret(
-			networkLosses.NaiveValue,                                // L^-_i
+			args.NetworkLosses.NaiveValue,                           // L^-_i
 			networkLossesByWorker.InfererLosses[infererLoss.Worker], // L_ij
 			lastRegret.Value,
-			alpha,
+			args.AlphaRegret,
 			blockHeight,
 		)
 		if err != nil {
 			return errorsmod.Wrapf(err, "Error computing and building inferer regret")
 		}
-		err = k.SetNaiveInfererNetworkRegret(ctx, topicId, infererLoss.Worker, newInfererRegret)
+		err = args.K.SetNaiveInfererNetworkRegret(args.Ctx, args.TopicId, infererLoss.Worker, newInfererRegret)
 		if err != nil {
 			return errorsmod.Wrapf(err, "Error setting inferer regret")
 		}
 	}
 
-	sort.Slice(networkLosses.OneOutInfererValues, func(i, j int) bool {
-		return networkLosses.OneOutInfererValues[i].Worker < networkLosses.OneOutInfererValues[j].Worker
+	sortedOneOutInfererValues := args.NetworkLosses.OneOutInfererValues
+	sort.Slice(sortedOneOutInfererValues, func(i, j int) bool {
+		return sortedOneOutInfererValues[i].Worker < sortedOneOutInfererValues[j].Worker
 	})
```

Comment:
same wrt sorting -- already sorteD?

Reply:
you are right, sorted in synth.CalcNetworkLosses